cli/src/commands-index.js

```diff
@@ -65,6 +66,7 @@ export default {
   repl,
   send,
   signMessage,
+  hash,
```

Comment:
i recommend a more specific name for this command, like `testZencashHash` or something. on the long term i'm not sure if it makes so much sense to have this command here, but maybe we could made a test later based on this idea.

Reply:
This command is generic, but we could rename it `testGetTrustedInputFromTxHash`, WDYT ?

Reply:
good to me!